### ISSUE TITLE
nvm 0.12.7 and 0.12.6 have sass issues

### ISSUE THREAD

**Issue by justin808** [justin808/react-webpack-rails-tutorial#52]
libsass has an issue. Maybe there's an update?

```
08:57:58 hot.1    | Error: `libsass` bindings not found in /Users/justin/a/react-webpack-rails-tutorial/client/node_modules/node-sass/vendor/darwin-x64-14/binding.node. Try reinstalling `node-sass`?
```
If anybody has a PR that fixes this issue...

Thanks in advance.

## Full message
```
foreman start -f Procfile.dev                                                                                                                                                                              ✹ [8:57:48]
08:57:56 web.1    | started with pid 46827
08:57:56 client.1 | started with pid 46828
08:57:56 hot.1    | started with pid 46829
08:57:57 client.1 | Webpack dev build for Rails
08:57:57 hot.1    | sleep: using busy loop fallback
08:57:57 hot.1    | Listening at localhost:3000...
08:57:58 hot.1    | Error: `libsass` bindings not found in /Users/justin/a/react-webpack-rails-tutorial/client/node_modules/node-sass/vendor/darwin-x64-14/binding.node. Try reinstalling `node-sass`?
08:57:58 hot.1    |     at Object.sass.getBinaryPath (/Users/justin/a/react-webpack-rails-tutorial/client/node_modules/node-sass/lib/extensions.js:148:11)
08:57:58 hot.1    |     at Object.<anonymous> (/Users/justin/a/react-webpack-rails-tutorial/client/node_modules/node-sass/lib/index.js:16:36)
08:57:58 hot.1    |     at Module._compile (module.js:460:26)
08:57:58 hot.1    |     at Object.Module._extensions..js (module.js:478:10)
08:57:58 hot.1    |     at Module.load (module.js:355:32)
08:57:58 hot.1    |     at Function.Module._load (module.js:310:12)
08:57:58 hot.1    |     at Module.require (module.js:365:17)
08:57:58 hot.1    |     at require (module.js:384:17)
08:57:58 hot.1    |     at Object.<anonymous> (/Users/justin/a/react-webpack-rails-tutorial/client/node_modules/sass-loader/index.js:4:12)
08:57:58 hot.1    |     at Module._compile (module.js:460:26)
08:57:58 hot.1    | Error: `libsass` bindings not found in /Users/justin/a/react-webpack-rails-tutorial/client/node_modules/node-sass/vendor/darwin-x64-14/binding.node. Try reinstalling `node-sass`?
08:57:58 hot.1    |     at Object.sass.getBinaryPath (/Users/justin/a/react-webpack-rails-tutorial/client/node_modules/node-sass/lib/extensions.js:148:11)
08:57:58 hot.1    |     at Object.<anonymous> (/Users/justin/a/react-webpack-rails-tutorial/client/node_modules/node-sass/lib/index.js:16:36)
08:57:58 hot.1    |     at Module._compile (module.js:460:26)
08:57:58 hot.1    |     at Object.Module._extensions..js (module.js:478:10)
08:57:58 hot.1    |     at Module.load (module.js:355:32)
08:57:58 hot.1    |     at Function.Module._load (module.js:310:12)
08:57:58 hot.1    |     at Module.require (module.js:365:17)
08:57:58 hot.1    |     at require (module.js:384:17)
08:57:58 hot.1    |     at Object.<anonymous> (/Users/justin/a/react-webpack-rails-tutorial/client/node_modules/sass-loader/index.js:4:12)
08:57:58 hot.1    |     at Module._compile (module.js:460:26)
08:57:58 web.1    | => Booting WEBrick
08:57:58 web.1    | => Rails 4.2.1 application starting in development on http://localhost:4000
08:57:58 web.1    | => Run `rails server -h` for more startup options
08:57:58 web.1    | => Ctrl-C to shutdown server
08:57:59 web.1    | [2015-07-24 08:57:59] INFO  WEBrick 1.3.1
08:57:59 web.1    | [2015-07-24 08:57:59] INFO  ruby 2.2.1 (2015-02-26) [x86_64-darwin14]
08:57:59 web.1    | [2015-07-24 08:57:59] INFO  WEBrick::HTTPServer#start: pid=46827 port=4000
08:57:59 hot.1    | webpack: wait until bundle finished: /comments.json
```

**Comment by justin808** [comment]
I tried deleting my node modules and reinstalling. There's a deeper issue here.

```
09:10:08 hot.1    | ERROR in ./~/css-loader!./~/sass-loader?imagePath=/assets/images!./~/bootstrap-sass-loader/bootstrap-sass-styles.loader.js!./bootstrap-sass.config.js
09:10:08 hot.1    | Module build failed:
09:10:08 hot.1    | // IMPORTANT: Make sure to keep the customizations defined in this file
09:10:08 hot.1    |        ^
09:10:08 hot.1    |       File to import not found or unreadable: Users/justin/a/react-webpack-rails-tutorial/client/node_modules/bootstrap-sass/assets/stylesheets/bootstrap/variables.scss
09:10:08 hot.1    |       in /Users/justin/a/react-webpack-rails-tutorial/client/bootstrap-sass.config.js (line 1, column 9)
09:10:08 hot.1    |  @ ./~/style-loader!./~/css-loader!./~/sass-loader?imagePath=/assets/images!./~/bootstrap-sass-loader/bootstrap-sass-styles.loader.js!./bootstrap-sass.config.js 4:14-218 13:2-17:4 14:20-224
```

**Comment by adavkay** [comment]
Hey Justin have you tried `npm rebuild node-sass` ?

**Comment by ssaunier** [comment]
Same problem for me with node `0.12.4`. Also, `npm rebuild node-sass` does not solve the problem.

**Comment by Johnnycus** [comment]
Same problem with node v0.10.33. npm rebuild node-sass doesn't solve problem. Deleting and reinstalling node_modules works, but only for first time.

**Comment by justin808** [comment]
This is fixed in PR #56. I'd love some help getting the tests to pass.
